src/test/resources/expected-classes/kt/AddLabelsToLabelableMutationResolver.kt.txt

```diff
@@ -0,0 +1,13 @@
+package com.github.graphql
+
+
+@javax.annotation.Generated(
+    value = ["com.kobylynskyi.graphql.codegen.GraphQLCodegen"],
+    date = "2020-12-31T23:59:59-0500"
+)
+interface AddLabelsToLabelableMutationResolver {
+
+    @Throws(Exception::class)
```

Comment:
Do we want this?

Reply:
Just for the same function as Java and scala

Reply:
This annotation only affects Java callers of the Kotlin code. So I'm not sure it matters much here.

Reply:
There are cross language possibilities. Just like us in Scala,.we only focus on Scala, but have Java users. So consider generating Java or Scala both need another language.